File: backend/src/main/java/ru/focsit/backend/dto/SongDto.java
package ru.focsit.backend.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SongDto {
    private Long id;
    private String name;
    private String genre;
    private String artist;
    private String album;
    private Integer duration;
}

File: mobile/app/src/main/java/ru/focsit/mobile/data/SongDto.kt
package ru.focsit.mobile.data

/**
 * Данные о песне.
 * Используется для представления информации о песне, включая её название, жанр, исполнителя и продолжительность.
 *
 * @property id Уникальный идентификатор песни.
 * @property name Название песни.
 * @property genre Жанр песни.
 * @property artist Исполнитель песни.
 * @property album Альбом, в который входит песня.
 * @property duration Продолжительность песни в секундах.
 */
data class SongDto(
    val id: Long? = null,
    val name: String,
    val genre: String,
    val artist: String,
    val album: String,
    val duration: Int
)